contrib/docker-compose/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     restart: always
     volumes:
       - /lib/modules:/lib/modules:ro
-      - run:/run
+      - /run:/run
```

Comment:
I will get rid of this ASAP, before dropping "WIP" from this PR, it's a temporary hack to make a socket available for k8s running from `./hack/local-up-cluster.sh`